src/common/soc.c

```diff
@@ -78,6 +78,16 @@ void fill_soc(struct system_on_chip* soc, char* soc_name, SOC soc_model, int32_t
   }
 }
 
+VENDOR try_match_soc_vendor_name(char* vendor_name)
```

Comment:
If would surround this with `#ifdef __WIN32` to make clear is only used when compiling for Windows.